### PR TITLE
making lims part of the app context

### PR DESCRIPTION
**Pull request by mayabrandi**

### CODE REVIEW

File: vogue/commands/load/sample.py [hunk 1]
```diff
@@ -28,12 +27,13 @@
 def sample(sample_lims_id, dry, many, load_from, new, date):
     """Read and load lims data for one ore all samples. When loading many smaples,
     the different options -f, -n, -d are used to delimit the subset of samples to load."""
+    
     try:
-        lims = Lims(BASEURI,USERNAME,PASSWORD)
+        lims = current_app.lims
```
Comment (northwestwitch) [inline vogue/commands/load/sample.py]: You don't need the try any more here, you could check if app.lims

Comment (mayabrandi) [reply vogue/commands/load/sample.py]: True! 👍

File: vogue/commands/load/sample.py [hunk 2]
```diff
@@ -28,12 +27,13 @@
 def sample(sample_lims_id, dry, many, load_from, new, date):
     """Read and load lims data for one ore all samples. When loading many smaples,
     the different options -f, -n, -d are used to delimit the subset of samples to load."""
-    try:
-        lims = Lims(BASEURI,USERNAME,PASSWORD)
-    except Exception:
+    
+    if not current_app.lims:
```
Comment (mayabrandi) [inline vogue/commands/load/sample.py]: Like that? @northwestwitch

**Comment by northwestwitch** [comment]
Yes, test it before merging maybe?

**Comment by mayabrandi** [comment]
Already tested it!

**Pull request merged**